src/main/java/Duke.java

```diff
@@ -37,342 +48,367 @@ public void run() throws DukeException {
-                        throw new DukeException("OOPS!!! The description of find cannot be empty");
-                    }
-                    String keyword = nextLine.substring(5);
-                    int len = inputs.size();
-                    for(int i = 0; i < len; i++) {
-                        Input input = inputs.get(i);
-                        if (input.content.contains(keyword)) {
-                            if (input.isDone) {
-                                System.out.println((i+1) + ". " + input.id + "[/] " + input.content +
-                                        input.printedTime);
-                            } else {
-                                System.out.println((i+1) + ". " + input.id + "[x] " + input.content +
-                                        input.printedTime);
-                            }
-                        }
-                    }
-                }
-                
-            
-            
-            void taskTodo(String nextLine) throws DukeException {
-                if (nextLine.equals("todo") || nextLine.equals("todo ")) {
-                    throw new DukeException("OOPS!!! The description of a todo cannot be empty.");
-                }
-                Todo todo = new Todo(nextLine.substring(5));
-                inputs.add(todo);
-                int count = inputs.size();
-                System.out.println("Got it. I've added this task: \n" + "  [T][x] " + todo.content +
-                        "\n Now you have " + count + " tasks in the list");
-                storage.writeToFile(inputs);
+            int numTaskDone = Integer.valueOf(nextLine.substring(5));
+            if (numTaskDone > inputs.size()) {
+                throw new DukeException("OOPS!!! Task does not exist.");
             }
+            System.out.println("Nice! I've marked this task as done:");
+            Input inputType = inputs.get(numTaskDone - 1);
+            inputType.taskDone();
+            System.out.println("[/] " + inputType.content + " " + inputType.printTime);
+        }
 
-            void taskDeadline(String nextLine) throws DukeException {
-                if (nextLine.equals("deadline") || nextLine.equals("deadline ")) {
-                    throw new DukeException("OOPS!!! The description of a deadline cannot be empty.");
-                }
-                int charLoc = nextLine.indexOf("/by");
-                Deadline deadline = new Deadline(nextLine.substring(9, charLoc), nextLine.substring(charLoc + 4));
-                inputs.add(deadline);
-                int count = inputs.size();
-                System.out.println("Got it. I've added this task: \n" + "  [D][x] " + deadline.content +
-                        deadline.printedTime + "\n Now you have " + count + " tasks in the list");
-                storage.writeToFile(inputs);
+        /*
+         * removes task, as specified by the user, from list
```

Comment:
Might be a little typo here, but don't forget to caps the first letter of the sentence!